sushi-config.yaml

```
@@ -16,7 +16,7 @@ license: 'CC-BY-4.0'
 jurisdiction: http://unstats.un.org/unsd/methods/m49/m49.htm#001 "World"
 publisher:
   name: HL7 International - FHIR Infrastructure WG
-  url: https://www.hl7.org/Special/committees/fiwg/index.cfm
+  url: http://hl7.org/Special/committees/fiwg
```

```suggestion
  url: https://hl7.org/Special/committees/fiwg
```

Is there any reason not to use TLS?